cli/commands/status.js

```diff
@@ -339,8 +339,8 @@ class ServicesStatus {
   }
 
   processArgs (args) {
-    var res = []
-    for (var i = 0; i + 1 < args.length; i++) {
+    const res = []
+    for (let i = 0; i + 1 < args.length; i++) {
       if (args[i].startsWith('--')) {
         res.push(args[i].substring(2, args[i].length) + '=' + args[i + 1])
       }
```

Comment:
is it okay to JS? const res; res.push(data)? It seems linter fixed this one automatically.

Reply:
It is valid JS... it is fine... `const` is only prevents the reassignment of the variable, but not the mutation of the object itself :laughing:

Reply:
Not sure is it constant array or constant address of array.

Reply:
It is constant address of array

Reply:
Resolved.